app/constants/errorMessages.js

```diff
@@ -3,8 +3,9 @@ const errorMessages = {
   invalidEmail: 'Email no válido',
   invalidEmailResource: 'Email no válido, no pertenece a Wolox',
   invalidPassword: 'Contraseña no válida',
+  invalidPasswordLength: 'La contraseña debe tener como minimo 8 caracteres',
   passwordRequired: 'Contraseña requerida',
-  nameRequired: 'Nombre requerido',
+  firstNameRequired: 'Nombre requerido',
   lastNameRequired: 'Apellido requerido',
   userAlreadyExist: 'Usuario existente'
 };
```

Comment:
Response and error messages are usually in English. No need to change it anyway, they are just string messages :smile: